src/main/java/uk/co/caprica/picam/CaptureFailedException.java

```diff
@@ -22,6 +22,7 @@
 /**
  * Exception thrown if a capture was requested but failed for some reason.
  */
+@SuppressWarnings({"unused", "WeakerAccess"})
```

Comment:
Same as before, what is it that is unused and what is it that can be weaker access?